helpers/constants.py

```diff
@@ -0,0 +1,21 @@
+# Git import constants
+git_import_branch="master"
+git_import_submitted_modal_title="Request submitted."
+git_import_repo_modules_count=10
```

Comment:
Add a comment here explicitly mentioning that: "this count specifies the number of modules being imported from 'git_import_repo_url' which is being used to verify if git import functionality imported all the modules successfully. You will have to change this number if you change the default repo used."